modules/gitops-release-notes-1-4-0.adoc

```diff
@@ -0,0 +1,55 @@
+
+
+
+* As an administrative user, when you give Argo CD access to a namespace by using the `argocd.argoproj.io/managed-by` label, it assumes namespace-admin privileges. These privileges are an issue for administrators who provide namespaces to non-administrators, such as development teams, because the privileges enable non-administrators to modify objects such as network policies.
++
+With the current release, administrators can configure a common cluster role for all the managed namespaces. In role bindings for the Argo CD application controller, the Operator refers to the `CONTROLLER_CLUSTER_ROLE` environment variable. In role bindings for the Argo CD server, the Operator refers to the `SERVER_CLUSTER_ROLE` environment variable. If these environment variables contain custom roles, the Operator does not create the default admin role. Instead, it uses the existing custom role for all managed namespaces.
+link:https://issues.redhat.com/browse/GITOPS-1290[GITOPS-1290]
+
+[id="fixed-issues-1-4-0_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Previously, when the Route to the {gitops-title} Application Manager (kam) was accessed without specifying a path in the URL, a default page without any helpful information was displayed to the user. The current update fixes this issue so that the default page displays download links for kam. link:https://issues.redhat.com/browse/GITOPS-923[GITOPS-923]
+
+* Previously, setting a resource quota in the namespace of the Argo CD custom resource might cause the setup of the Red Hat SSO (RH SSO) instance to fail. The current release fixes this issue by setting a minimum resource request for the RH SSO deployment pods. link:https://issues.redhat.com/browse/GITOPS-1297[GITOPS-1297]
+
+* Previously, if you changed the log level for the `argocd-repo-server` workload, the Operator didn't reconcile this setting. The workaround was to delete the deployment resource so that the Operator recreated it with the new log level. With this update, the log level is correctly reconciled for existing `argocd-repo-server` workloads. link:https://issues.redhat.com/browse/GITOPS-1387[GITOPS-1387]
+
+* Before this update, if the Operator managed an Argo CD instance that lacked the `.data` field in the `argocd-secret` Secret, the Operator on that instance crashed. This update fixes the issue so that the Operator doesn't crash when the `.data` field is missing. Instead, the secret regenerates and the `gitops-operator-controller-manager` resource is redeployed.
+link:https://issues.redhat.com/browse/GITOPS-1402[GITOPS-1402]
+
+* Previously, the `gitopsservice` service was annotated as an internal object. This update removes the annotation so you can update or delete the default Argo CD instance and run GitOps workloads on infrastructure nodes by using the UI. link:https://issues.redhat.com/browse/GITOPS-1429[GITOPS-1429]
```

Comment:
Anyone, would you help me identify which UI this release note is referring to?

Reply:
It is the OpenShift DevConsole.